templates/projects/answerfieldsconfig.html

```diff
@@ -13,18 +13,18 @@
             <fields-config > </fields-config>
         </div>
         <div>
-            <redundancy-config :redundancy-config="{{redundancy_config}}"></redundancy-config>
+            <redundancy-config :redundancy-config="{{consensus_config}}"></redundancy-config>
```

Comment:
tag name also to be updated matching config type here and other places?

Reply:
follow-up jira to handle the config objects mapped to correct html tags. Thanks!